tools/tracegen/StringUtils.cpp

```diff
@@ -44,8 +45,13 @@ StringUtils::startsWithUpperLower(const char *text, const char *prefix)
 const char *
 StringUtils::containsUpperLower(const char *text, const char *toFind)
 {
+#if !defined(OMRZTPF)
 	while ('\0' != *text) {
 		if (tolower(*text) == tolower(*toFind)) {
+#else /* !defined(OMRZTPF) */
+	while ('\0' != (unsigned char)*text) {
```

Comment:
I recompiled the code undoing the z/TPF mod and compiled it cleanly.  Not sure why we originally had that mod (maybe at the time a different compiler?).  Anways, because it was the only mod we had in StringUtils.cpp  I removed it from the commit.